<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>templatesetup</class>
 <widget class="QMainWindow" name="templatesetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>899</width>
    <height>612</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>899</width>
    <height>612</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>899</width>
    <height>612</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="maximumSize">
           <size>
            <width>645</width>
            <height>520</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QGroupBox
{
border-radius: 5px;
border-width: 1px;
border-style: solid;
border-color: darkCyan;
}

QGroupBox::title
{
subcontrol-origin: margin;
subcontrol-position: top center;
padding: 0 3px;
}</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QCustomPlot" name="widgetPaint" native="true">
             <property name="minimumSize">
              <size>
               <width>625</width>
               <height>500</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>625</width>
               <height>500</height>
              </size>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="MoveWidgetControl" name="widgetControl" native="true">
                <property name="minimumSize">
                 <size>
                  <width>625</width>
                  <height>500</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>625</width>
                  <height>500</height>
                 </size>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_9">
                 <item>
                  <spacer name="verticalSpacer">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>479</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QGroupBox
{
border-radius: 5px;
border-width: 1px;
border-style: solid;
border-color: darkCyan;
}

QGroupBox::title
{
subcontrol-origin: margin;
subcontrol-position: top center;
padding: 0 3px;
}</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonOk">
               <property name="text">
                <string>确定</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushButtonCancel">
               <property name="text">
                <string>取消</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox
{
border-radius: 5px;
border-width: 1px;
border-style: solid;
border-color: darkCyan;
}

QGroupBox::title
{
subcontrol-origin: margin;
subcontrol-position: top center;
padding: 0 3px;
}</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QPushButton" name="pushButtonAdd">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>  添加模板</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>:/image/image/plus.ico</normaloff>:/image/image/plus.ico</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButtonSub">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>  删除模板</string>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>:/image/image/delete.ico</normaloff>:/image/image/delete.ico</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="labelTemplateTitle">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color:#2c3e50;
color:white;

QLabel
{
    height:40px;
    font-size:28px;
}</string>
               </property>
               <property name="text">
                <string>模板名称</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListWidget" name="listWidgetTemplate">
               <property name="styleSheet">
                <string notr="true">QListWidget::item
{
    height:40px;
    font-size:20px;
}

QListWidget::item::hover
{
    background-color:rgb(9,150,200,100);
}

QListWidget::item:selected
{
    background-color:rgb(9,150,200,255);
}</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelTitle">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color:#2c3e50;

color:white;


QLabel
{
    height:40px;
    font-size:28px;
}</string>
               </property>
               <property name="lineWidth">
                <number>1</number>
               </property>
               <property name="text">
                <string>工具箱</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QListWidget" name="listWidgetControl">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>160</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QListWidget::item
{
    height:40px;
    font-size:20px;
}

QListWidget::item::hover
{
    background-color:rgb(9,150,200,100);
}

QListWidget::item:selected
{
    background-color:rgb(9,150,200,255);
}</string>
               </property>
               <property name="lineWidth">
                <number>1</number>
               </property>
               <item>
                <property name="text">
                 <string>文字</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>病理号</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>条码</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>二维条码</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QtTreePropertyBrowser" name="widgetProperty" native="true"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDelete"/>
  </widget>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../res.qrc">
     <normaloff>:/image/image/critical.png</normaloff>:/image/image/critical.png</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="toolTip">
    <string>删除</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtTreePropertyBrowser</class>
   <extends>QWidget</extends>
   <header>qttreepropertybrowser.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>printtemplate/qcustomplot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MoveWidgetControl</class>
   <extends>QWidget</extends>
   <header>printtemplate/movewidgetcontrol/movewidgetcontrol.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
